swift/ql/lib/codeql/swift/generated/Callable.qll

```diff
@@ -7,37 +7,70 @@ import codeql.swift.elements.decl.ParamDecl
 
 module Generated {
   class Callable extends Synth::TCallable, Element {
+    /**
+     * Gets the self param, if it exists.
+     * This is taken from the "hidden" AST and should only be used to be overridden by classes.
```

Comment:
I'm not especially fond of the spacing here.  I think I'd prefer it to be all on the same line, or with a blank line between.

Reply:
Ok, I was hesitating on the blank line as well